native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -1325,6 +1428,7 @@ static void update_workers_lbstatus(proxy_server_conf *conf, apr_pool_t *pool, s
     /* update lbstatus if needed */
     for (i = 0; i < size; i++) {
         nodeinfo_t *ou;
+        int cont;
```

Comment:
Why here? and not in the if (child_stopping) { }?

Reply:
if () {
    int count;
    ....
}
is OK

if () {
    ...
    int count;
   ...
}
is NOT.

Reply:
Well, it turned out I was wrong :) Fixed, thank you @jfclere !